developer_setup.md

```diff
@@ -2,156 +2,151 @@
+   | yum  | `sudo yum -y install openssl-devel readline-devel zlib-devel` |
+   | apt  | `sudo apt -y install libssl-dev libreadline-dev zlib1g-dev` |
+   | brew | `brew install openssl` |
 
-In order to compile Ruby, native Gems and native NodeJS modules, other libraries are required.
+3. In addition, in order to compile Ruby, native Gems and native NodeJS modules, other libraries are required.
 
-|      |     |
-| ---- | --- |
-| dnf  | `sudo dnf -y install @c-development libffi-devel postgresql-devel libxml2-devel libcurl-devel cmake python` |
-| yum  | `sudo yum -y install @development libffi-devel postgresql-devel libxml2-devel libcurl-devel cmake python` |
-| apt  | `sudo apt -y install build-essential libffi-dev libpq-dev libxml2-dev libcurl4-openssl-dev cmake python` |
-| brew | `brew install cmake` |
+   |      |     |
+   | ---- | --- |
+   | dnf  | `sudo dnf -y install @c-development libffi-devel postgresql-devel libxml2-devel libcurl-devel cmake python` |
+   | yum  | `sudo yum -y install @development libffi-devel postgresql-devel libxml2-devel libcurl-devel cmake python` |
+   | apt  | `sudo apt -y install build-essential libffi-dev libpq-dev libxml2-dev libcurl4-openssl-dev cmake python` |
+   | brew | `brew install cmake` |
 
-### Prerequisite services
+### Services
 
 ManageIQ requires a memcached instance for session caching and a PostgreSQL database for persistent data storage.
 
 #### memcached
 
-##### Install
+1. Install
 
-|            |     |
-| ---------- | --- |
-| dnf        | `sudo dnf -y install memcached` |
-| yum        | `sudo yum -y install memcached` |
-| apt        | `sudo apt -y install memcached` |
-| brew       | `brew install memcached` |
-| containers | N/A |
+   |            |     |
+   | ---------- | --- |
+   | dnf        | `sudo dnf -y install memcached` |
+   | yum        | `sudo yum -y install memcached` |
+   | apt        | `sudo apt -y install memcached` |
+   | brew       | `brew install memcached` |
+   | containers | N/A |
 
-##### Start the service
+2. Start the service
 
-|            |     |
-| ---------- | --- |
-| systemd    | `systemctl enable --now memcached` |
-| brew       | `brew services start memcached` |
-| containers | `podman run --detach --publish 11211:11211 memcached` |
+   |            |     |
+   | ---------- | --- |
+   | systemd    | `systemctl enable --now memcached` |
+   | brew       | `brew services start memcached` |
+   | containers | `podman run --detach --publish 11211:11211 memcached` |
 
 #### PostgreSQL
 
-##### Install
+1. Install
 
-|            |     |
-| ---------- | --- |
-| dnf        | `sudo dnf -y install postgresql-server` |
-| yum        | `sudo yum -y install postgresql-server` |
-| apt        | `sudo apt -y install postgresql` |
-| brew       | `brew install postgresql` |
-| containers | N/A |
-
-###### NOTE
+   |            |     |
+   | ---------- | --- |
+   | dnf        | `sudo dnf -y install postgresql-11` |
```

Comment:
This doesn't work for me on Fedora 33

Reply:
Ah yeah seems like you can't install a class of versions like you can on deb, probably best to leave this as just `dnf -y install postgresql` then @bdunne ?

Reply:
Yes, `sudo dnf -y install postgresql-server`.  I assume it will be the same for `yum` as well.

Reply:
Ok I put back the postgresql-server for dnf and yum